core/modules/file/file.pages.inc

```diff
@@ -553,6 +555,17 @@ function file_manage_form($form, &$form_state, File $file) {
     '#default_value' => file_uri_scheme($file->uri),
   );
 
+  if (!empty($file->uri)) {
+    $form['destination']['path'] = array(
+      '#type' => 'textfield',
```

Comment:
**Technically** this is a URL - not a path. Any reason why this cannot be `'#type' => 'markup'`, or even better:

```suggestion
      '#type' => 'item',
```

https://api.backdropcms.org/form_api#item

>**Description**: Generate a display-only form element allowing for an optional title and description.

Reply:
why item? Does it still provide an easy-to-copy-paste input field?

Reply:
I tried item, and the input disappeared entirely. I don't think this is a supported FAPI element?

Reply:
Pushed a new PR